### PR TITLE
sample name can be a number or a character and it will not crush at c…

### PR DESCRIPTION
**Pull request by danilodileo**
…ollect stats step

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

This PR deals with #151

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -378,9 +378,7 @@ process {
 
     withName: SUM_EGGNOG {
         publishDir = [
-        path: { "${params.outdir}/summary_tables" },
-        mode: 'copy',
-        pattern: "eggnog_summary.tsv"
+            enabled: false
```
Comment (erikrikarddaniel) [inline conf/modules.config:381]: Are these intermediary files that should not be saved? (Same below.)

File: modules/local/merge_summary_tables.nf [hunk 2]
```diff
@@ -31,9 +31,9 @@ process MERGE_TABLES {
     library(tidyr)
     library(stringr)
 
-    summary <- data.frame(sample = character(), database = character(),field = character(),  value = numeric(), stringsAsFactors = FALSE) %>%
+    summary <- data.frame(sample = character(), database = character(), field = character(), value = numeric(), stringsAsFactors = FALSE) %>%
         union(list.files(pattern = "*.tsv.gz") %>%
-        map_df(~read_tsv(.,  show_col_types  = TRUE))) %>%
+            map_df(~ read_tsv(., show_col_types = TRUE) %>%  mutate(sample = as.character(sample)))) %>%
```
Comment (erikrikarddaniel) [inline modules/local/merge_summary_tables.nf]: Indentation seems off, should be the same as `union()` above and `pivot_wider()` below.

**Review by erikrikarddaniel — APPROVED** [review]
A small indentation thing, plus I'm not sure what's going on in `modules.conf` with the saving. As I suspect you know better than I, I'm approving anyway.

File: conf/modules.config (hunk 1 above)
Comment (danilodileo) [reply conf/modules.config:381]: yes, we decided to not have them in the folder as they are with all other info in overall_stats.tsv.gz

File: modules/local/merge_summary_tables.nf (hunk 2 above)
Comment (danilodileo) [reply modules/local/merge_summary_tables.nf]: the two map_df are inside union function, so both should be indent.

**Pull request merged**